config/opa-service/rego/anubis_management_api_policy_test.rego

```diff
@@ -6,7 +6,7 @@ policy_data = {
 "user_permissions": {
     "admin@mail.com": [
       {
-        "action": "acl:Read",
+        "action": "acl:Control",
```

Comment:
why from Control to read?

Reply:
You need `acl:Control` to alter an entity's policies, so that's what it's testing there.